client/ts-web-component/package.json

```diff
@@ -1,20 +1,25 @@
 {
-  "name": "replicache-todo-ts-component",
+  "name": "replicache-quickstarts-todo-ts-component",
   "private": true,
   "version": "0.0.0",
   "type": "module",
   "scripts": {
     "lint": "eslint --ext .ts,.tsx,.js,.jsx .",
     "dev": "vite",
     "build": "npm run lint && tsc && vite build",
```

Comment:
perhaps tsc --noEmit, since I think it is just for checking types and vite does the actual transpile.